src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import ProctoredExamSettings from './ProctoredExamSettings';
+
+const defaultProps = {
+  courseId: 'course-v1%3AedX%2BDemoX%2BDemo_Course',
+};
+
+describe('ProctoredExamSettings', () => {
+  it('updates settings based on proctoring provider', () => {
+    const component = mount(<ProctoredExamSettings {...defaultProps} />);
+
+    // confirm that changing proctoring provider to proctortrack changes zendesk ticket field
+    component.find('select#proctoringProvider').simulate('change', { target: { value: 'proctortrack' } });
+    expect(component.find('input#createZendeskTickets').prop('checked')).toEqual(false);
```

Comment:
Can you please move this to a different test case instead of cramp it into this single one?